src/config/config.go

```diff
@@ -53,9 +54,9 @@ func ValidDateConfiguration(config *Configuration) {
 	}
 
 	if fileInputL == "live" {
-		//require prefix and collector
-		if len(config.Connector) == 0 || len(config.Prefix) == 0 {
-			fmt.Println("Choosing live data input stream requires to input at least one value for the connector and at lease one value for the prefix")
+		//require at least 1 subscription
+		if len(config.Subscriptions) == 0 {
+			fmt.Println("Choosing live data input stream requires to input at least one subscription")
```

Comment:
Since the system will do nothing with no subscriptions, it should probably exit. Probably cleanest to define an error condition for this function and return an Error with the information here so that `main()` can print it and then `os.Exit()` with a non-zero status.